packages/expo-application/ios/EXApplication/EXApplication.m

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+#import <UMCore/UMUtilities.h>
+#import <EXApplication/EXApplication.h>
+#import <UIKit/UIKit.h>
+
+@interface EXApplication ()
+
+
+@end
+
+@implementation EXApplication
+
+UM_EXPORT_MODULE(ExpoApplication);
+
+- (dispatch_queue_t)methodQueue
+{
+  return dispatch_get_main_queue();
+}
+
+- (NSString *)appleIFV {
+  if(NSClassFromString(@"UIDevice") && [UIDevice instancesRespondToSelector:@selector(identifierForVendor)]) {
+    return [[UIDevice currentDevice].identifierForVendor UUIDString];
+  }
+  return nil;
+}
+
+- (NSString *) applicationName
```

Comment:
just inline all these methods, easy enough to understand without the extra lines of code